crates/telio-wg/src/adapter/boring.rs

```diff
@@ -7,7 +7,7 @@ use std::net::{IpAddr, Ipv4Addr};
 use std::sync::Arc;
 use std::{io, ops::Deref};
 use telio_crypto::PublicKey;
-use telio_utils::telio_log_debug;
+use telio_utils::{telio_log_debug, telio_log_generic};
```

Comment:
I really don't like that now we have to have a magical depenency, it would be better if you use $crate::telio_log_generic and avoid this issue.

Same is allso a good way to rexpose all trace innteranls in same create, to avoid needing aditional tracing rexposes